cloud-pubsub/deployment/pubsub.yaml

```diff
@@ -1,3 +1,18 @@
+# Copyright 2020 Google LLC
+#
+# Licensed under the Apache License, Version 2.0 (the "License");
+# you may not use this file except in compliance with the License.
+# You may obtain a copy of the License at
+#
+#     http://www.apache.org/licenses/LICENSE-2.0
+#
+# Unless required by applicable law or agreed to in writing, software
+# distributed under the License is distributed on an "AS IS" BASIS,
+# WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+# See the License for the specific language governing permissions and
+# limitations under the License.
+
+# [START gke_pubsub_deployment]
```

Comment:
one thing to think about: do we have a programmatic way to make sure our region tag names are unique? 
let's assume that this repo is the only source of truth for GKE samples - can always circle back and re-address with all our repos.

Reply:
I don't think we have a programmatic way in place. My workflow is always to check sample tracker as the source of truth before creating a new region tag, but let me know if you have any other ideas to guarantee uniqueness.